docker-compose.yml

```diff
@@ -77,19 +77,17 @@ services:
       timeout: 1s
       retries: 5
 
-  caddy:
+  nginx:
     <<: *common
-    image: caddy:latest
+    image: nginx:latest
     restart: always
     ports:
       - "80:80"
-      - "443:443"
     networks:
       - harvest
     volumes:
-      - ./shared/config/caddy:/etc/caddy
-      - caddy_data:/data
-      - caddy_config:/config
+      - ./shared/config/nginx:/etc/nginx
```

Comment:
Let's specify the exact conf file instead.

It's showing the `conf.d` directory in vscode because of the volumes.

Reply:
60bdc52c5459c0a040af85ef775a18debdd0f59d